src/components/CreateVmWizard/steps/_TableInlineEditRow.js

```diff
@@ -4,12 +4,13 @@ import TableConfirmButtonsRow from 'patternfly-react/dist/js/components/Table/Ta
 
 const TableInlineEditRow = props => {
   const buttonsPosition = (window, rowDimensions) => {
+    const { rowHeight } = props
```

Comment:
As far as I can tell, `rowHeight` is not provided as a prop to the component, it is only being defined in the `defaultProps` as the constant value 31.  This will force the buttons to render in one spot no matter the height of the row.

Screen shot of the component tree in master (i.e. without this change):
![VM Portal - Google Chrome_034](https://user-images.githubusercontent.com/3985964/95791129-b1194c00-0cae-11eb-880f-67c414ab59ca.png)